src/applications/claims-status/components/ClaimsContentionList.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useRef } from 'react';
+import PropTypes from 'prop-types';
+import { setFocus } from '../utils/page';
+
+const MAX_CONTENTIONS = 3;
+
+function ClaimsContentionList({ contentions, contentionsHeader }) {
+  const ellipsis = useRef(null);
+  const additionalContentions = useRef(null);
+  const viewMoreBtn = useRef(null);
+  const hasContentions = contentions && contentions.length;
+  const contentionsText = hasContentions
+    ? contentions
+        .slice(0, MAX_CONTENTIONS)
+        .map(cond => cond.name)
+        .join(', ')
+    : 'Not available';
```

Comment:
We could probably use a guard statement to just return `Not available` when there are no contentions. This would enable us to remove checks for `hasContentions` below